classes/class-date-interval.php

```diff
@@ -7,12 +7,6 @@
 
 namespace WP_Stream;
 
-// Load Carbon to Handle dates much easier.
-if ( ! class_exists( 'Carbon\Carbon' ) ) {
-	require_once wp_stream_get_instance()->locations['inc_dir'] . 'lib/Carbon.php';
-}
-use Carbon\Carbon;
```

Comment:
@delawski Props for removing the dependency!

Reply:
Thanks! I noticed there were already a few bug reports caused by Carbon (which was not added via Composer and thus hard to update). Since the dates handling is rather light in the plugin, I went with the native API.